R/is_empty.R

```diff
@@ -65,8 +65,8 @@ is_empty <- function(x, first.only = TRUE, all.na.empty = TRUE) {
       zero_len <- nchar(x) == 0
       # return result for multiple elements of character vector
       if (first.only) {
-        zero_len <- .is_true(zero_len[1])
-        if (length(x) > 0) x <- x[!is.na(x)][1]
+        zero_len <- .is_true(zero_len[!is.na(x) & !is.null(x)][1])
```

Comment:
Not sure if this would work as intended? It will find the first non-missing element, but it should definitely check for the very first element.

Reply:
I can remove the condition `!is.na(x) & !is.null(x)` from lines 68 and 69 if you like. 

I had introduced it because there was an error when computing `frq` of character vectors whose first element was `NA`. I believe this was fixed by other ways, and with the other changes in this PR (introduce `first.only = FALSE` in line 242 of `frq.r`) it should not be a problem any more.

On the other hand, since I have never fully understood which is the profit of check only the first element of a character vector, so I thought that what has really interest is the first non-missing element.

So, as I told you at the beginning, if you ask me I remove this condition.

Reply:
> I believe this was fixed by other ways

Yes, I think so, too. I would suggest reverting this change in lines 68/69, as it might break code in other packages.

Reply:
Done!